packager.sh

```diff
@@ -601,9 +599,8 @@ celery_wait() {
     return 1
 }
 
-sleep 30
-sudo supervisorctl status
-sudo supervisorctl start openquake-celery
+sudo systemctl status openquake\\*
```

Comment:
``openquake-\\*`` seems to me more strict and self-explaining about what kind of services you could find related to openquake.